tests/Doctrine/Tests/DBAL/Schema/DB2SchemaManagerTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Schema;
+
+use Doctrine\Common\EventManager;
+use Doctrine\DBAL\Configuration;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver;
+use Doctrine\DBAL\Platforms\DB2Platform;
+use Doctrine\DBAL\Schema\DB2SchemaManager;
+
+/**
+ * @covers \Doctrine\DBAL\Schema\DB2SchemaManager
+ */
+final class DB2SchemaManagerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Connection
+     */
+    private $conn;
+
+    /**
+     * @var DB2SchemaManager
+     */
+    private $manager;
+
+    protected function setUp()
+    {
+        $eventManager = new EventManager();
+        $driverMock = $this->getMock(Driver::class);
```

Comment:
The problem is [that we're depending on an old PHPUnit version](https://github.com/doctrine/dbal/blob/1b3c36b4550d4e29d2a896e032d7432d411682d1/composer.json#L19).